build/toolchain/tizen/tizen_toolchain.gni

```diff
@@ -30,6 +30,10 @@ template("tizen_toolchain") {
   _tizen_toolchain_prefix =
       "${tizen_sdk_root}/tools/arm-linux-gnueabi-gcc-9.2/bin/arm-linux-gnueabi-"
 
+  if (_invoker_toolchain_args.current_cpu == "arm64") {
```

Comment:
Add `if` for the "arm" as well, and in the `else` block print error message that the CPU architecture is not supported.